keen/keen-js
# Update mocha-phantom-js, add in sinon and requirejs to package.json

## What does this PR do?

Previously, fresh installs of keen-js would give the following errors:

```
npm WARN mocha-phantomjs@3.6.0 requires a peer of phantomjs@1.9.1 - 1.9.7-15 but none was installed.
```
and
```
├── UNMET PEER DEPENDENCY requirejs@^2.1.0
├── UNMET PEER DEPENDENCY sinon@*
```

This PR updates `mocha-phantom-js` to the newest version to take care of the first error message, and adds `requirejs` and `sinon` to `package.json` to take care of the second error message.

## How to test

* Open the `keen-js` project.
* Delete `node_modules`
* Run `npm install`
* Make sure those error messages don't appear.

I am running npm `3.8.3` and node `v5.10`. Please make sure you're upgraded to the latest versions of npm and node.
